erica/request_processing/erica_input/v2/possibly_aliased_input.py

```diff
@@ -0,0 +1,8 @@
+from humps import camelize
+from pydantic import BaseModel
+
+
+class PossiblyAliasedInput(BaseModel):
```

Comment:
I think we should rename this, because it is now not aliased anymore, but camelizable or something along the lines of it^^
Also: We might not even need the ` allow_population_by_field_name` option anymore.

Reply:
Good idea, I will rename it to CamelCaseInput.

We do need `allow_population_by_field_name` though since otherwise snake-cased fields won't be parsed.